setup.py

```diff
@@ -50,6 +50,7 @@
     entry_points={"console_scripts": ["qdarkstyle=qdarkstyle.__main__:main"]},
     extras_require=extras_require,
     install_requires=install_requires,
+    include_package_data=True,
```

Comment:
May we need to change the MANIFEST to include the resources too. In the link I've posted it suggests using general rules such as (not sure about ):

graft qdarkstyle
graft example
graft test
prune .*
prune docs
include *.py *.rst *.ini *.cfg *.scss *.svg *.qrc *.qss
global-exclude *.py[cod] __pycache__ *.so

Not element names, which must be updated if file names are changed.